Generate docs (#2356)
kyma-project/cli: docs/user/gen-docs/01-130-kyma_alpha_module_manage.md

# kyma alpha module manage

> Manage module

Reviewer: ```suggestion
Sets the module to the managed state.
```

## Synopsis

> Use this command to manage an existing module

Reviewer: ```suggestion
Use this command to set an existing module to the managed state.
```

```bash
kyma alpha module manage <module> [flags]
```

## Flags

```bash

>       --policy string   Set custom resource policy (Possible values: CreateAndDelete, Ignore) (default "CreateAndDelete")

Reviewer: ```suggestion
      --policy string   Sets a custom resource policy (Possible values: CreateAndDelete, Ignore) (default "CreateAndDelete")
```

```

## Flags inherited from parent commands

```bash
  -h, --help                    Help for the command
      --kubeconfig string       Path to the Kyma kubeconfig file
      --show-extensions-error   Print possible error when fetching extensions failed
```

## See also

> * [kyma alpha module](kyma_alpha_module.md) - Manage kyma modules

Reviewer: ```suggestion
* [kyma alpha module](kyma_alpha_module.md) - Manages kyma modules.
```
